src/components/radio-button-group/RadioButtonGroup.tsx

```diff
@@ -18,18 +18,18 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-import * as RadioGroup from '@radix-ui/react-radio-group';
 import styled from '@emotion/styled';
+import * as RadioGroup from '@radix-ui/react-radio-group';
 import { ReactNode } from 'react';
 
 interface Props {
-  onChange?: (value: string) => void;
-  options: RadioOption[];
-
   // Group Props
   defaultValue?: string;
   disabled?: boolean;
+
   id: string;
+  onChange?: (value: string) => void;
+  options: RadioOption[];
   required?: boolean;
   value?: string;
 }
```

Comment:
This breaks the semantic separation that I introduced. Maybe we shouldn't enforce prop ordering?

Reply:
Ah yes, good catch, I'll drop the rule